orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -56,7 +57,11 @@ def _forward(self, *inputs, **kwargs):
         # Get the module that flattens both input and output
         self._flattened_module = _io._FlattenedModule(self._original_module)
 
-        self._execution_manager = GraphExecutionManagerFactory(self._flattened_module)
+        onnx_export_type = torch.onnx.OperatorExportTypes.ONNX
+        if 'onnx_export_type' in kwargs:
+            onnx_export_type = kwargs['onnx_export_type']
```

Comment:
More a question for the front-end team re API design... but is there a reason to prefer this approach over having a default onnx_export_type in the definition above?

Reply:
We have revised API design based on offline discussion with frontend team in commit 140. Frontend team suggests using `onnx_export_type`, not `kwargs`. Also, the `onnx_export_type` is not exposed through `ORTModule`'s constructor. It will be added later following torch_ort pattern.